### PR TITLE
Remove eslint disable

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
This is mostly just reordering a few huge files so that functions aren't used before they're defined, and also removed eslint-disable comments from them. I also replaced `ts-disable` with `ts-expect-error`, which doesn't require an eslint override and also will throw an error if, in the future, it is no longer needed. (Redundant `ts-expect-error` statements generate an error if there is no error)

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
Everything still works thank you

File: lib/actions/form.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// TODO: Typescript
```
Comment (philip-cline) [inline lib/actions/form.js:1]: Should the Typescript TODO stand?

**Review by philip-cline — APPROVED** [review]
Just one quick question, works and looks good to me.

File: lib/actions/form.js (hunk 1 above)
Comment (daniel-heppner-ibigroup) [reply lib/actions/form.js:1]: Good catch!

**Pull request merged**